frame/nomination-pools/src/lib.rs

```diff
@@ -639,7 +731,11 @@ impl<T: Config> BondedPool<T> {
 			// destroying it cannot switch states, so by being in destroying we are guaranteed no
 			// other delegators can possibly join.
 			(_, true) => {
-				ensure!(target_delegator.points == self.points, Error::<T>::NotOnlyDelegator);
+				// TODO: a secondary invariant here is that no delegator can have 0 points.
```

Comment:
This should just be a sanity check, if it doesn't already exist

Reply:
2b52a2582bf8882527e093fd2dab0f14abec2d9e